components/UI/Drawer/components/channels/twitter/PlaceholderTweet.tsx

```diff
@@ -1,13 +1,21 @@
 import Typography from "@mui/material/Typography";
 import TwitterLogo from "./TwitterLogo";
 import { FeedChannelTwitterProps, TwitterParameters } from "../../types";
+import { Chip } from "@mui/material";
 
 type Props = {
   source: TwitterParameters;
+  reason: string;
   full_text?: FeedChannelTwitterProps["full_text"];
 };
 
-const PlaceholderTweet = ({ source, full_text }: Props) => {
+const PlaceholderTweet = ({ source, reason, full_text }: Props) => {
+  const isNotNan = (value: any) => {
+    return !!value && value !== "NaN" && value !== "nan";
```

Comment:
Very small enhancement, we can convert last 2 conditions to
`value.toString().toLowerCase() !== "nan"`

If the PR is urgent, please ignore the comment. Good job!

Reply:
Done! @cemiluzunhasan